data/human/boarding missions.txt

```diff
@@ -406,3 +406,344 @@ mission "Pirate Mutiny"
+				decline
+
+			label "blackmailing alone"
+			`"It'll be a cold day in hell before I get blackmailed by someone dumb enough to board a disabled ship all alone," sneers Tarquin.`
+				goto "attack alone but tarquin is prepared"
+
+			label "the direct approach"
+			branch "acting stupid alone" "attack with a big boarding party"
+				"flagship crew" < 3
+
+			label "acting stupid alone"
+			`Captain Tarquin throws back his head and laughs at the exact moment that he draws a concealed pistol and shoots you with it five times - all in the blink of an eye. Your last thoughts concern the effectiveness of his distraction and the foolishness of your bravado.`
+				die
+
+			label "bad luck"
+			`Quick as a flash, he whips out a concealed pistol and drills a hole in your spine before you can react. The dangerous game you were playing ends abruptly.`
+				die
+
+			label "good luck"
+			`Quick as a flash, he whips out a concealed pistol, but you're faster. It's the last mistake he ever makes. You search the body of ex-captain Tarquin, finding several concealed knives in addition to his handgun, the credit chip he was going to pay you, and a tiny unlocking device. You go to rescue the crew with it.`
+				goto "rescue the crew"
+
+			label "rescue the crew"
+			`	The ship's crew quarters have been fashioned into a brig with stout cell doors. Fortunately the unlocking device opens them all, and close to a dozen malnourished wretches stumble out, telling you the whole story: captain Tarquin has been keeping them as slaves for the past several months, locking them in their quarters when the ship makes berth, and with no food as a punishment when they complain. They are quite happy to see his lifeless body sprawled out on the deck plating.`
+			`	You turn the <origin> over to them and return to your ship.`
+				accept
+
+			label "rescue the crew with tarquin captured"
+			`	The ship's crew quarters have been fashioned into a brig with stout cell doors. Fortunately the unlocking device opens them all, and close to a dozen malnourished wretches stumble out, telling you the whole story: captain Tarquin has been keeping them as slaves for the past several months, locking them in their quarters when the ship makes berth, and with no food as a punishment when they complain. They are quite happy to hear that he has been taken into captivity.`
+			apply
+				set "captured tarquin"
+			`	You turn the <origin> over to them and return to your ship to figure out what to do with Tarquin.`
+			`	A quick check of the public data in the galactic criminal justice database doesn't reveal that he's wanted for any crimes. You decide to turn him to Republic law enforcement on Earth.`
+				accept
+
+	on decline
+		payment 1000000
+	on accept
+		payment 350000
+
+mission "Turn in Tarquin"
+	description "Deliver captain Tarquin to law enforcement on <destination>."
+	passengers 1
+	to offer
+		has "captured tarquin"
+	destination "Earth"
+	on complete
```

Comment:
I believe it would be better to make this an "on offer" which instantly declines. Not sure if it will make any change though

Reply:
Good idea, I'll tweak it.